src/pages/Photos/components/PhotosList/PhotoList.scss

```diff
@@ -0,0 +1,44 @@
+.photo-list {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  padding-top: $top-menu-height;
+  min-height: 100vh;
+
+  &__card {
```

Comment:
Please note that the elements should be vertically aligned on the right side
<img width="1033" alt="Screenshot 2023-05-03 at 7 28 27 PM" src="https://user-images.githubusercontent.com/39587602/235947004-6a72c65c-6927-4966-b872-22974d2cd0d7.png">